scripts/junit2issue.py

```diff
@@ -41,11 +41,11 @@ class ZephyrTestCase(TestCase):
     '''
 
     GITHUB_ISSUE_COMMENTS_TEMPLETE = '''
-    Also fails on ${PLATFORM} for ${ZEPHYR_VERSION}
+Also fails on ${PLATFORM} for ${ZEPHYR_VERSION}
     '''
 
     GITHUB_ISSUE_TITLE_TEMPLETE = '''
-    tests-ci : ${MODULE}: ${SUBMODULE}: ${CASE_NAME} test ${RESULT}
+tests: ${MODULE}: ${SUBMODULE}: ${CASE_NAME} test ${RESULT}
```

Comment:
you probably want to remove the newlines.

Reply:
@nashif  updated, please check again